rpms/README.md

```diff
@@ -5,53 +5,74 @@ needed to build a set of RPMs that are useful for working with Geotrellis.  If
 you are not intending to alter the composition or versions of the RPM
 packages, then you should be able to use the published binaries described in
 the README at the root of this project and do not need to build these
```

Comment:
There is no description of pre-built RPMs in the root README.  This sentence should be fixed.

Reply:
Done - it looks like that was removed at some point.